docs/src/stories/Design System/icons/Icons.tsx

```diff
@@ -0,0 +1,808 @@
+import "./icons.css"
+import * as icons from "components/icons"
+import { useDarkMode } from "storybook-dark-mode"
+
+function Header() {
+  return (
+    <div className="h-[49px] bg-grey-800 -mx-[25px] -mt-[16px] items-center flex flex-row">
+      <div className="mx-[8px] mt-[5.5px] mb-[9.5px]">
+        <icons.AdaptiveLogo />
+      </div>
+      <div className="mt-[18.5px] mb-[18.5px]">
+        <p className="title text-white">Icons</p>
+      </div>
+    </div>
+  )
+}
+
+function getTextColorClass() {
+  return useDarkMode() ? "text-grey-50" : "text-grey-900"
+}
+function getIconColorClass() {
+  return useDarkMode() ? "fill-grey-300" : "fill-grey-600"
+}
+function getCaptionColorClass() {
+  return useDarkMode() ? "text-grey-200 italic" : "text-grey-600 italic"
+}
+
+const purposeText = {
+  title: "Purpose",
+  description: [
+    "Icons are more than decoration. They draw attention and add clarity to areas and actions in our screen designs.",
+    "Use them with purpose:",
+    "They are a supplement, not a substitute, for onscreen language. Using an icon without a label to go with it should be rare.",
+    "Your choice to use an icon, and your choice of icon, should have a specific user or brand goal.",
+    "Consistent icon usage can create useful patterns that the user can learn.",
+  ],
+}
+function Purpose({
+  title,
+  description,
+}: {
+  title: string
+  description: string[]
+}) {
+  return (
+    <div>
+      <div className="icons-purpose-title">
+        <p className={getTextColorClass() + " title"}>{title}</p>
+      </div>
+      <div className="icons-purpose-body">
+        <p className={getTextColorClass() + " body mb-[16px]"}>
+          {description[0]}
+        </p>
+        <p className={getTextColorClass() + " body font-bold"}>
+          {description[1]}
+        </p>
+        <div className="ml-[20px]">
+          <ul className="list-disc">
+            <li className={getTextColorClass() + " body"}>{description[2]}</li>
+            <li className={getTextColorClass() + " body"}>{description[3]}</li>
+            <li className={getTextColorClass() + " body"}>{description[4]}</li>
+          </ul>
```

Comment:
```suggestion
          <ul className={`list-disc ${getTextColorClass()} body`}>
            <li>{description[2]}</li>
            <li>{description[3]}</li>
            <li>{description[4]}</li>
          </ul>
```
Since the children are all the same, you can put it on the parent and the children will inherit it. save you from rewriting it multiple time